pkgs/tools/networking/ripe-atlas-tools/default.nix

```diff
@@ -0,0 +1,95 @@
+{ lib
+, python3
+, fetchFromGitHub
+, installShellFiles
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "ripe-atlas-tools";
+  version = "3.0.2";
+
+  src = fetchFromGitHub {
+    owner = "RIPE-NCC";
+    repo = "ripe-atlas-tools";
+    rev = "v${version}";
+    sha256 = "sha256-5AMqBXxJZOtI0/2NrEjrUfNXWKc7sn6kZX26766LBUM=";
+  };
+
+  postPatch = ''
+    # This mapping triggers network access on docs generation: https://github.com/RIPE-NCC/ripe-atlas-tools/issues/235
+    sed -i '/^intersphinx_mapping/d' docs/conf.py
+    # TODO: Ensure user-agent is picked up during build, remove me when https://github.com/RIPE-NCC/ripe-atlas-tools/pull/236
+    echo "include ripe/atlas/tools/user-agent" >> MANIFEST.in
+  '';
+
+  nativeBuildInputs = with python3.pkgs; [
+    sphinx-rtd-theme
+    sphinxHook
+    installShellFiles
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    ripe-atlas-cousteau
+    ripe-atlas-sagan
+    ujson
+    ipy
+    python-dateutil
+    requests
+    tzlocal
+    pyyaml
+    pyopenssl
+  ];
+
+  preBuild = ''
+    echo "RIPE Atlas Tools [NixOS ${lib.trivial.version}] ${version}" > ripe/atlas/tools/user-agent
```

Comment:
is the lib.trivial.version truly needed because now the hash of the derivation changes with every new commit
```
nix-repl> lib.trivial.version
"22.11.git.f01fe92ef7a"
```

```
$ ./maintainers/scripts/rebuild-amount.sh  --print HEAD~2
Estimating rebuild amount by counting changed Hydra jobs (parallel=unset).
      1 x86_64-darwin
      4 x86_64-linux

hadoop2.x86_64-linux                                                                                                    /nix/store/jks22jcyrjq0dv1cv9r94s0r0jb8g71g-hadoop-2.10.2
hadoop_3_2.x86_64-linux                                                                                                 /nix/store/dd43739y90v092q75njcpkmymc2wyfij-hadoop-3.2.4
nixos-install-tools.x86_64-linux                                                                                        /nix/store/mik0l9v9qkd7sxflhyf5x1y12vhrpzfc-nixos-install-tools-22.11.git.f01fe92ef7a
ripe-atlas-tools.x86_64-darwin                                                                                          dist=/nix/store/ygnvmrglykq01jnnc9sk3ckra3fy1y2v-ripe-atlas-tools-3.0.2-dist;/nix/store/7sp2maasq6dcvp4cabdd1p8370yzxgqs-ripe-atlas-tools-3.0.2
ripe-atlas-tools.x86_64-linux                                                                                           dist=/nix/store/kfvmchwhcxdkangjb1dn15x7nry4qlkz-ripe-atlas-tools-3.0.2-dist;/nix/store/zfv63zfknp0jlmnr1h2b3zv1v7g3az79-ripe-atlas-tools-3.0.2
```

Would `lib.versions.majorMinor lib.trivial.version` suffice

```
nix-repl> lib.versions.majorMinor lib.trivial.version
"22.11"
```

Reply:
Good catch! This should be dropped.

Reply:
#196059